sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Demo/Program.cs

```diff
@@ -65,6 +65,7 @@ private static async Task GenerateTelemetry()
                 Console.WriteLine("Request");
                 using (var activity = s_activitySource.StartActivity("Request", kind: ActivityKind.Server))
                 {
+                    await Task.Delay(100 + _random.Next(0, 100));
```

Comment:
could this be a random value without the default value of 100?
I'm hesitant to introduce delays into this loop.